dev/public/static/js/diagram-navigation.js

```diff
@@ -24,6 +24,8 @@ function configureFitAndZoomButtons() {
   Object.values(ZoomType).forEach(zoomType => {
     document.getElementById(`zoom-${zoomType}`).onclick = () => zoom(zoomType);
   });
+  document.getElementById('lock-navigation').onclick = () => disableNavigation();
+  document.getElementById('unlock-navigation').onclick = () => enableNavigation();
```

Comment:
**nitpick:** can you use enable/disable in ids instead of lock/unlock.
I get the idea, but using the same wording in the identifiers and API makes it more straightforward IMHO.

Reply:
fixed ...